stringfuzz/transformers/reverse.py

```diff
@@ -0,0 +1,31 @@
+'''
+Reversing every string literal
+'''
+
+from stringfuzz.ast import StringLitNode, ConcatNode, ReConcatNode
+from stringfuzz.ast_walker import ASTWalker
+
+__all__ = [
+    'reverse',
+]
+
+class ReverseTransformer(ASTWalker):
+    def __init__(self, ast):
+        super(ReverseTransformer, self).__init__(ast)
+
+    @property
+    def ast(self):
+        return self._ASTWalker__ast
```

Comment:
Sorry I only just noticed this, but it feels like a really sketchy practice to "know" how Python's name mangling works. Also, the `ast` property is repeated verbatim across all transformers. What do you think of modifying `walk()` to just return its AST? That would also let you avoid the `return transformer.ast` at the end of every transformation function.

Reply:
I would really like that. I'll do it now.